experimental/SetPartitions/src/SetPartitions.jl

```diff
@@ -19,11 +21,16 @@ import Oscar:
     parent,
     degree,
     tensor_product,
-    @req
+    @req,
+    RingElement,
+    scale,
```

Comment:
This Import together with the export below may export some oscar functionality that was be thought of to be kept internal. But as written above, I think your method scale shouldn't be there at all, and then you can get rid of both import and export as well